applications/luci-app-advanced-reboot/Makefile

```diff
@@ -0,0 +1,20 @@
+# Copyright (c) 2017 Stan Grishin (stangri@melmac.net)
+# This is free software, licensed under the GNU General Public License v3.
+
+include $(TOPDIR)/rules.mk
+
+PKG_LICENSE:=GPL-3.0+
+PKG_MAINTAINER:=Stan Grishin <stangri@melmac.net>
+
+LUCI_TITLE:=Advanced Linksys Reboot Web UI
+LUCI_DESCRIPTION:=Provides Web UI (found under System/Advanced Reboot) to reboot supported Linksys routers to\
+	an altnerative partition. Also provides Web UI to shut down (power off) your device. 	Supported dual-partition\
+	routers are listed at https://github.com/stangri/openwrt-luci/blob/luci-app-advanced-reboot/applications/luci-app-advanced-reboot/README.md.
```

Comment:
You should likely remove the dot from the end of the link (after README.md)
....-advanced-reboot/README.md. 

Otherwise copy-pasting that link will be hard (and automatic link parsing might include dot.

(and don't worry about the LEDE forum link, yet. We can fix that if it ever gets necessary)